home/ivp-planning/user-interface/the-3d-panel.md

```diff
@@ -1,28 +1,31 @@
-| ![](../../../.gitbook/assets/iVP\_icon\_3D\_FOV\_settings.jpg)                     | Changes the field of view of the currently chosen camera                                                                                                                                     |
-| ![](../../../.gitbook/assets/iVP\_icon\_3D\_reset\_camera\_view.jpg)               | Resets the camera view to a centered default perspective                                                                                                                                     |
-| ![](../../../.gitbook/assets/iVP\_icon\_3D\_add\_current\_camera\_view\_point.jpg) | Adds a new [virtual camera](../advanced-tools/virtual-cameras.md)                                                                                                                            |
-| ![](../../../.gitbook/assets/iVP\_icon\_3D\_update\_selected\_camera.jpg)          |                                                                                                                                                                                              |
+| ---------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| 
+ ![](../../../.gitbook/assets/iVP\_icon\_3D\_toggle_camera_mode.png)                                | Toggles the camera mode
+| ![](../../../.gitbook/assets/iVP_icon_3D_selected_view.jpg)              | Allows you to switch between different [virtual cameras](virtual-camera-panel.md)                                                                                                            |
+| ![](../../../.gitbook/assets/iVP_icon_3D_move.png)                              | Switches to a mode that makes it possible to [move](../machines/selecting-and-moving-objects.md) the selected object(s) by using the mouse|
+| ![](../../../.gitbook/assets/iVP_icon_3D_rotate.png)                      | Switches to a mode that makes it possible to [rotate](../machines/scale-and-rotate-objects.md#rotate-objects) the selected object(s)|
+| ![](../../../.gitbook/assets/iVP\_icon\_3D\_scale.png)                             | Switches to a mode that makes it possible to [scale](../machines/scale-and-rotate-objects.md#scale-objects) the selected object(s)|
+| ![](../../../.gitbook/assets/iVP_icon_3D_global_space.png)          |  "Global space" refers to a coordinate system that isn't tied to any particular object. Its axes remain fixed in direction, regardless of any changes made to an object                                                                                                                                                                                         |
+|  ![](../../../.gitbook/assets/iVP_icon_3d_field_of_view.png)          |  Lets you adjust the Field of View in the 3D-View
```

Comment:
This is more precise. We should also stay consistent with calling it a _panel_ instead of a _view_. Please make sure this is the case everywhere on this page.

```suggestion
|  ![](../../../.gitbook/assets/iVP_icon_3d_field_of_view.png)          |  Lets you adjust the Field of View of the 3D panel's camera.
```